### ISSUE TITLE
[BUG] On pressing view more, many comments and replies become disjointed and become their own separate comment

### ISSUE THREAD

**Issue by GetPsyched6** [laurencedawson/sync-for-reddit#592]
**Describe the bug**
On a post with a good number of comments where we can press view more > atleast 50% of comments seen after pressing view more are fully disjointed, as in the replies to a parent comment, the op's reply to the parent comment and sometimes even the view more button of a specific parent-child comment thread... becomes their own separate comment, making it very confusing to follow the comments of a post

Picture 1

![20210226_003844](https://user-images.githubusercontent.com/3417276/109215517-b2b25f00-77cc-11eb-8b0b-783b194aa4fa.jpg)

Picture 2

![20210226_004125](https://user-images.githubusercontent.com/3417276/109215574-c65dc580-77cc-11eb-870a-f56a34e65d91.jpg)

Picture 3

![20210226_004056](https://user-images.githubusercontent.com/3417276/109215550-bcd45d80-77cc-11eb-91d5-05b7b70b67e7.jpg)

**To Reproduce**
Test post thread> Daily Discussion of r/soccer
(Pressing view more)
Continue scrolling to new comments

**Smartphone (please complete the following information):**
 - Device: Note 9
 - OS: Android 10
 - Browser Chrome
 - Version Beta 35

**Comment by laurencedawson** [comment]
If they're disjointed that's how they're represented in the API

**Comment by GetPsyched6** [comment]
Oh, that's unfortunate :( the comment section after pressing view more kinda becomes a free for all 😅 makes it so hard to follow what everybody's trying to say

But still, thanks for addressing the bug thread! :)
